CHANGELOG.md

```diff
@@ -11,8 +11,27 @@
   [Marcelo Fabri](https://github.com/marcelofabri)
   [#1387](https://github.com/realm/SwiftLint/issues/1453)
 
+* Add `modificationDate(forFileAtPath:)` function requirement to
+  `LintableFileManager` protocol.
+  [Victor Pimentel](https://github.com/victorpimentel)
+
+* Replace `configurationHash` parameters in all `LinterCache` initializers
+  with `configurationDescription` parameters.
+  [Victor Pimentel](https://github.com/victorpimentel)
+
+* Remove `fileHash` parameter from `LinterCache.cache(violations:forFile:)`
+  and `LinterCache.violations(forFile:)` functions.
+  [Victor Pimentel](https://github.com/victorpimentel)
+
+* Remove `hash` attribute from `Configuration` struct.
+  [Victor Pimentel](https://github.com/victorpimentel)
+
 ##### Enhancements
 
+* Cache linter results for files unmodified since the previous linter run.
+  [Victor Pimentel](https://github.com/victorpimentel)
+  [#1184](https://github.com/realm/SwiftLint/issues/1184)
```

Comment:
new computer accidentally has `whitespace = fix` in its `.gitconfig` so it automatically trimmed trailing whitespace when rebasing. Promptly removed that.